evap/evaluation/models.py

```diff
@@ -692,6 +692,11 @@ def can_publish_rating_results(self):
         # the rating results are only published if at least the configured number of participants voted during the evaluation for anonymity reasons
         return self.num_voters >= settings.VOTER_COUNT_NEEDED_FOR_PUBLISHING_RATING_RESULTS
 
+    @cached_property
+    def percent_voted(self):
+        return -(self.num_voters / self.num_participants)
```

Comment:
The `-` here seems wrong - I would say that the ordering is also the wrong way around right now